monitoring/mock_ridsp/routes_ridsp.py

```diff
@@ -1,30 +1,88 @@
+import arrow
 import datetime
+from typing import List, Optional
 
 import flask
+import s2sphere
 
-from monitoring.monitorlib import rid
+from monitoring.monitorlib import geo, rid
+from monitoring.monitorlib.rid_automated_testing.injection_api import TestFlight
+from monitoring.monitorlib.typing import StringBasedDateTime
```

Comment:
Nit: StringBasedDateTime takes both string and datetime, recommend to rename

Reply:
StringBasedDateTime is fundamentally a representation of a datetime, and the way it represents that datetime is as a string.  It can be constructed from multiple input formats, but it is not its the formats it can accept.  With this in mind, what would you suggest as a new name?

Reply:
(will merge for now but address suggestions in a future PR)

Reply:
Dont really have a good suggest but perhaps `SemanticDateTime` ?